docs/source/migrating.rst

```diff
@@ -0,0 +1,189 @@
+Migrating from IPython
+======================
+
+`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+has changed a few things, moving the various language-agnostict
+components of IPython under the Jupyter umbrella. This doc should help
+you find what's changed, and how to keep doing what you've been doing.
+
+Where have all my files gone?
+-----------------------------
+
+AKA Why isn't my configuration having any effect anymore?
+
+Jupyter splitting out from IPython means that the locations of some
+files have moved, and Jupyter projects have not inherited everything
+from how IPython did it.
+
+The ``jupyter migrate`` command copies several of these files to their
+new homes.
+
+Quick reference
+~~~~~~~~~~~~~~~
+
+A quick reference for common locations of relocated files.
+
+Configuration files
+^^^^^^^^^^^^^^^^^^^
+
+-  ``~/.ipython/profile_default/static/custom`` → ``~/.jupyter/custom``
+-  ``~/.ipython/profile_default/ipython_notebook_config.py`` →
+   ``~/.jupyter/jupyter_notebook_config.py``
+-  ``~/.ipython/profile_default/ipython_nbconvert_config.py`` →
+   ``~/.jupyter/jupyter_nbconvert_config.py``
+-  ``~/.ipython/profile_default/ipython_qtconsole_config.py`` →
+   ``~/.jupyter/jupyter_qtconsole_config.py``
+-  ``~/.ipython/profile_default/ipython_console_config.py`` →
+   ``~/.jupyter/jupyter_console_config.py``
+
+``JUPYTER_CONFIG_DIR`` env can be used if ``~/.jupyter`` is not desired.
+
+Data files
+^^^^^^^^^^
+
```

Comment:
There is still the issue of custom profile, we might want to expand `jupyter migrate` to take a profile name and a target dir. But probably for another PR.